frontend_tests/node_tests/templates.js

```diff
@@ -24,3 +28,8 @@ run_test("or", () => {
     const html = require("../../static/templates/test/or.hbs")(args);
     assert.equal(html, "\n<p>last or</p>\n");
 });
+
+run_test("rendered_markdown", () => {
+    const html = require("../../static/templates/test/rendered_markdown.hbs")();
+    assert.equal(html, "example_rendered_markdown\n");
+});
```

Comment:
I think I'd prefer a test here that actually verifies at least one of the transformations this makes (perhaps even just `clean_user_content_links`).